module/forest/leveled_forrest_test.go

```diff
@@ -258,6 +259,24 @@ func TestLevelledForest_GetVertex(t *testing.T) {
 	assert.False(t, exists)
 }
 
+// TestLevelledForest_GetSize tests that GetSize returns valid size when adding and pruning vertices
```

Comment:
Nice tests!

would be better to add more:
1. Adding duplicated nodes again would not change the size. We can add the same 10 nodes again, and check the size didn't change. Would be better to implement in a separate test case rather than in the existing `TestLevelledForest_GetSize`.
2. Pruning twice with the same height will not change size, because nothing was pruned.